.solhint.json

```diff
@@ -2,9 +2,14 @@
   "extends": "solhint:recommended",
   "plugins": ["prettier"],
   "rules": {
+    "no-unused-import": "error",
     "compiler-version": "off",
     "max-states-count": "off",
     "no-empty-blocks": "off",
-    "func-visibility": ["warn",{"ignoreConstructors":true}]
+    "func-visibility": ["warn",{"ignoreConstructors":true}],
+    "custom-errors": "off",
+    "reason-string": "off",
+    "explicit-types": "off",
```

Comment:
I think we should fix these ones too instead of ignoring. Usually they come from the fact that one uses uint instead uint256